readme.txt

```diff
@@ -25,22 +23,21 @@ It's important to note that if you use meta fields to add extra text, images or
 
 [See Facebook's documentation for full details of the submission process.](https://developers.facebook.com/docs/instant-articles)
 
-Facebook requires a minimum number of articles in your feed before they will review it. Once your feed has been approved, you can set the constant `INSTANT_ARTICLES_LIMIT_POSTS` to `true` to limit the feed to only show posts that have been modified within the last 24 hours. (Facebook will ignore any articles which were last modified more than 24 hours ago.)
-
-Facebook will fetch your feed every 3 minutes.
+Facebook requires a minimum number of articles in your feed before they will review it. Once your feed has been approved, new posts will automatically be taken live on Instant Articles, and existing posts will be taken live once you update them.
 
 == Installation ==
 
-1. Upload the folder to the `/wp-content/plugins/` directory
-1. Activate the plugin through the 'Plugins' menu in WordPress
+1. Run `composer install` on the root of the plugin folder. Make sure you have [Composer](https://github.com/composer/composer) installed.
+2. Upload the folder to the `/wp-content/plugins/` directory
```

Comment:
Shouldn't it be opposite - upload, then run?
Or, "run composer on an environment where it is installed".